hacker_news/models.py

```diff
@@ -0,0 +1,25 @@
+from django.db import models
+
+
+class New(models.Model):
```

Comment:
Thats just for consistency with Django admin. Otherwise it shows Newss in Django admin